core-bundle/src/Resources/contao/templates/analytics/analytics_google.html5

```diff
@@ -4,7 +4,7 @@
 $GoogleAnalyticsId = 'UA-XXXXX-X';
 
 // DO NOT EDIT ANYTHING BELOW THIS LINE UNLESS YOU KNOW WHAT YOU ARE DOING!
-if ('UA-XXXXX-X' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
```

Comment:
Are you sure this is correct? `hasAuthenticatedBackendUser()` only checks `hasBackendUser()` and not `isPreviewMode()`.

Reply:
I think this is correct. The current implementation does not output the analytics code if _either_ you are logged in to the backend _or_ frontend preview is enabled. Which never makes sense, since you (currently) cannot use frontend preview without being logged in to the back end. With https://github.com/contao/contao/pull/3872 I think it is correct to track frontend-preview-only visitors, but certainly never users logged in to the back end.

Reply:
> I think it is correct to track frontend-preview-only visitors

Not sure about this. @contao/developers WDYT?

Reply:
I think I agree with @aschempp

Reply:
Front end preview visitors should definitely not be tracked in my opinion.